src/processElement.js

```diff
@@ -7,12 +7,14 @@ const processElement = element => new Promise((resolve, reject) => {
 
   if (el.type === 'text') {
     const text = $(el).text();
-    const firstChar = text.substr(0, 1);
-    if (['*', '-'].indexOf(firstChar) > -1) {
-      resolve(`\\${text}`);
-    } else {
-      resolve(text);
-    }
+    const escaped = text.replace(/[#_*~><>/\\[\]!()`-]+/g, (symbol) => {
+      let output = '';
+      for (let i = 0; i < symbol.length; i += 1) {
+        output += `\\${symbol[i]}`;
+      }
+      return output;
```

Comment:
I think something like this would be more elegant than using a for loop
```js
const escaped = text
  .replace(...), symbol =>
    symbol.split('').map(s => `\\${s}`))
  .replace(..., '')
```